src/test/regress/expected/distributed_procedure_0.out

```diff
@@ -0,0 +1,206 @@
+SET citus.next_shard_id TO 20030000;
+CREATE USER procedureuser;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
+SELECT run_command_on_workers($$CREATE USER procedureuser;$$);
+      run_command_on_workers       
+-----------------------------------
+ (localhost,57637,t,"CREATE ROLE")
+ (localhost,57638,t,"CREATE ROLE")
+(2 rows)
+
+CREATE SCHEMA procedure_tests AUTHORIZATION procedureuser;
+CREATE SCHEMA procedure_tests2 AUTHORIZATION procedureuser;
+SET search_path TO procedure_tests;
+SET citus.shard_count TO 4;
+-- Create and distribute a simple function
+CREATE OR REPLACE PROCEDURE raise_info(text)
+    LANGUAGE PLPGSQL AS $proc$
+BEGIN
+    RAISE INFO 'information message %', $1;
+END;
+$proc$;
+ERROR:  syntax error at or near "PROCEDURE"
+LINE 1: CREATE OR REPLACE PROCEDURE raise_info(text)
```

Comment:
shall we do something like here: https://github.com/citusdata/citus/blob/master/src/test/regress/expected/pg12_1.out